scripts/vipers/build_parent_sample.py

```diff
@@ -78,6 +78,8 @@ def extract_data(filename):
     # Loop through the header keys and add them to the results dictionary
     for key in HEADER_KEYS:
         results[key] = float(header[key])
+        if key == "REDSHIFT":
+            results["z"] = float(header[key])
```

Comment:
This way we still have `result["REDSHIFT"]` as an entry. If it is not used nor wanted I suggest the following change.
```suggestion
        if key == "REDSHIFT":
            results["z"] = float(header[key])
        else:
            results[key] = float(header[key])
```